### PR TITLE
[SC-201] Utility script to manage EC2s

### PR DESCRIPTION
**Pull request by zaro0508**
Script to stop EC2 instances when SC user buget is exceeded.

### CODE REVIEW

File: utils/manage_ec2.py [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+        description=__doc__)
+    parser.add_argument(
+        'synapse_id',
+        help='The synapse user ID')
+    parser.add_argument(
+        '-y', '--yes',
+        action='store_true',
+        help='Skip verification prompts')
+    parser.add_argument(
+        '-s', '--shutdown',
```
Comment (brucehoff) [inline utils/manage_ec2.py:24]: I would use the term 'stop' rather than 'shutdown'

**Comment by zaro0508** [comment]
> An admin will run this to delete all the instances of a user?
Yes @xschildw , either an Admin or the community manager will run this script

> @zaro0508, I think we also need the ability to terminate provisioned EC2 products through the servicecatalog API.
What is your rational behind this thinking @tthyer ?

**Comment by tthyer** [comment]
> What is your rational behind this thinking @tthyer ?
@zaro0508 The origin of the need for this script is the workflow where:
1. a user exceeds their quota
2. the community manager removes the user from the team that grants access
3. the community manager opens a JIRA ticket to remove the user's provisioned products that are EC2s
4. IT runs a script to remove those products

The question is whether the CM could sometimes also just ask you to shut things down rather than remove them -- I think it would be great to have that flexibility. But in the original requirements discussions, we talking about removing.

**Comment by zaro0508** [comment]
Hmm, I thought we discussed stopping.  Removing is a bit harsh don't you think @tthyer ?  I think the more common use case is to temporarily stop instances until we can get a response or payment from the user or team that's racking up all the expense?

**Comment by tthyer** [comment]
@zaro0508, it's not what was discussed when I was gathering the requirements, but if it's the way you want to manage this problem, that's fine. It is probably an acceptable short-term if you assume that we'll have actual cost management in the long run.

**Comment by ahayden** [comment]
I did try to terminate an instance through the communitymgr role yesterday and got permission denied.

**Comment by zaro0508** [comment]
> I did try to terminate an instance through the communitymgr role yesterday and got permission denied.

Good point @ahayden however I think that's a separate permissions issue.  We will need to adjust the community manager permissions if we want to allow them run this script.  Please create an issue or we can deal with when time comes.

**Pull request merged**